OOP/Restaurant/item.py

```diff
@@ -0,0 +1,164 @@
+"""Module Contains classes Item, OrderedITem, Stock, Order, Payment  and Delivery from a 
+restaurant classes representation"""
+
+from restaurant import Customer
+from employee import Employee
+
+class Item:
+    """Class representing the single item from menu"""
+    def __init__(self, category,item_name, price) -> None:
+        self.item_name = item_name
+        self.price=price
+        self.category=category
+        self.item_dic = None
+
+    def add_item(self, menu):
+        """Adds an item to menu"""
+        self.item_dic ={self.category:{self.item_name:self.price}}
+        menu.add_menu(self.item_dic)
+
+class OrderedItem(Item):
+    """Class representing an Item ordered by a customer"""
+
+    def __init__(self,date: str ,item: Item, quantity: int, customer: Customer) -> None:
```

Comment:
Here you added type hints, but other classes methods modules are missing type hints